### ISSUE TITLE
CSS Tooltip

### ISSUE THREAD

**Issue by oyeanuj** [davidchin/react-input-range#64]
Hi @davidchin! Firstly, thank for you building this useful component and putting out there!

It seems to be a common use-case for sliders to have some sort of a tooltip. I was wondering if adding a tooltip (css-only, so no new dependencies) is on your radar, or if you had any thoughts on about it?

Thank you!

**Comment by oyeanuj** [comment]
@davidchin To clarify, you have labels that show the progress, etc. I'm curious if it can be configured to show only on hover (min, max and hover).

Also, updated the title to be more accurate.

**Comment by oyeanuj** [comment]
@davidchin Any thoughts on this one?

**Comment by davidchin** [comment]
Yeah, that's a good idea. I'd make it an option though so people can opt in rather than have the tooltips showing by default.

**Comment by siarheipashkevich** [comment]
Hi @davidchin, do you have any updates?
